src/games/chess/board.tsx

```diff
@@ -51,7 +51,11 @@ export class Board extends React.Component<IBoardProps, {}> {
     }
     if (this.props.ctx.gameover) {
       return (
-        <GameLayout gameOver={this._getGameOver()} />
+        <GameLayout
+          gameOver={this._getGameOver()}
+          playAgain={((this.props.gameArgs) && ((this.props.gameArgs.mode === GameMode.AI)
```

Comment:
I don't think this logic should be here. Just send the gameArgs.mode to GameLayout and add this logic there instead.

Reply:
The logic should be in Game over tbh